src/main/java/io/stargate/sgv2/jsonapi/service/operation/tables/AlterTableAttempt.java

```diff
@@ -92,25 +90,5 @@ public enum AlterTableType {
     ADD_COLUMNS,
     DROP_COLUMNS,
     UPDATE_EXTENSIONS;
-
```

Comment:
This can stay, it is the OperationAttempt making sure it is correctly constructed - without worrying about whatever checks the caller has done

Reply:
solved, thanks